nemo/collections/asr/models/classification_models.py

```diff
@@ -364,7 +365,8 @@ def transcribe(
         Generate class labels for provided audio files. Use this method for debugging and prototyping.
 
         Args:
-            audio: (a single or list) of paths to audio files or a np.ndarray audio sample. \
+            audio: (a single or list) of paths to audio files or a np.ndarray audio array.
```

Comment:
Add `torch.dataloader` to `audio` types in the func signature.

Reply:
Done